index.js

```diff
@@ -344,6 +372,52 @@ class Help {
   }
 }
 
+class Argument {
+  /**
+   * Initialize a new command argument with the given name and description.
+   * The default is that the argument is required, and you can explicitly
+   * indicate this with <> around the name. Put [] around the name for an optional argument.
+   *
+   * @param {string} name
+   * @param {string} [description]
+   */
+
+  constructor(name, description) {
+    this.description = description || '';
+    this.variadic = false;
+
+    switch (name[0]) {
+      case '<': // e.g. <required>
+        this.required = true;
+        this._name = name.slice(1, -1);
+        break;
+      case '[': // e.g. [optional]
+        this.required = false;
+        this._name = name.slice(1, -1);
+        break;
+      default:
+        this.required = true;
+        this._name = name;
+        break;
+    }
+
+    if (this._name.length > 3 && this._name.slice(-3) === '...') {
+      this.variadic = true;
+      this._name = this._name.slice(0, -3);
+    }
+  }
```

Comment:
looks like this is needed as BC
```js
this.term = this._name; // legacy, BC
```

Reply:
That will be due to recent change to `visibleArguments()`, so returns `Argument` rather than plain object with `{term, description}`.

https://github.com/tj/commander.js/pull/1490/commits/f54d1edce8c7b77afd779b13c2ce2d38d331085f

Reply:
yes

Reply:
Not planning to patch a `term` property into Argument, as following the Help pattern there is now full `argumentTerm()`  method.